training/training.pages.inc

```diff
@@ -0,0 +1,228 @@
+<?php
+/**
+ * @file
+ * Page callbacks for the training module.
+ */
+
+/**
+ * Rendering example form.
+ *
+ * @param array $arg
+ *  Id of example form.
+ *
+ * @return array|mixed
+ *   The form array.
+ */
+function training_example_form_callback($arg) {
+  return drupal_get_form($arg);
+}
+
+/**
+ * Rendering list of links on other theme pages.
+ *
+ * @return string
+ *   String contents HTML tags for custom layout.
+ */
+function training_theme_html() {
+  $array = array(
```

Comment:
don't use special words and language structures as variable name 
replace it with $items, variable name is a noun that describes clearly and in short manner its value.